multicast/hear.py

```
@@ -244,6 +252,238 @@
+		Testcase 1: McastServer should be automatically imported.
+
+			>>> multicast.hear.McastServer.__name__ is not None
+			True
+			>>> multicast.hear.McastServer.__log_handle__ is not None
+			True
+			>>>
+
+	"""
+
+	def __init__(
+		self, server_address: tuple, RequestHandlerClass: type, bind_and_activate: bool = True
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)